kidney.html

```diff
@@ -55,40 +55,43 @@ <h2>Menu</h2>
       <section id="wrapper">
         <header id="kidney-header">
           <div class="inner">
-            <h2>PREDICTING CHRONIC KIDNEY DISEASE</h2>
+            <h2>Prediciendo la enfermedad crónica de rinión</h2>
           </div>
         </header>
 
         <!-- Content -->
         <div class="wrapper">
           <div class="inner">
             <section>
-              <h3 class="major">The issue:</h3>
+              <h3 class="major">El problema:</h3>

```

Comment:
Nico, veo que el texto de algunos de estos `<h3>` tienen dos puntos (`:`) y otros no. ¿Te parece bien si hacemos que todos no lo tengan? La otra es que todos lo tengan. Yo te voy dejando sugerencias para removerlos, pero si te parece mejor dejarlos, ignoralas nomás y asegurate de que queden todos con el `:`. 
```suggestion
              <h3 class="major">El problema</h3>
```